### DISCUSSION TITLE
Welcome to recipe-buddy Discussions!

### DISCUSSION THREAD

**Discussion by georgegebbett** [georgegebbett/recipe-buddy#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by R3SPONSIV3-ONE** [comment]
Drop me a short email on db@oosterid.com 

I could help you with the FrontEnd

Regards, 
David Oosterveer
